setup.py

```diff
@@ -35,7 +35,7 @@
         "Topic :: Terminals"],
     packages=find_packages() + ['axiom.plugins', 'xmantissa.plugins'],
     install_requires=[
-        "twisted>=14.0.0",
+        "twisted>=14.0.0,<16.4.0",
```

Comment:
It might be http://twistedmatrix.com/trac/ticket/8843? In any case, there should be a comment explaining the issue as best it is understood (ideally linking to a ticket, but even just the comment from the commit would be sufficient).